packages/vuetify/src/components/VRating/VRating.tsx

```diff
@@ -204,7 +205,7 @@ export default defineComponent({
               : iconProps.label ? <span>{iconProps.label}</span>
               : <span>&nbsp;</span>
             }
-            {slots.item ? slots.item(iconProps) : (
+            { slots.item ? slots.item(iconProps) : (
```

Comment:
what's our rule for space after `{` or not in JSX? You didn't add it for the prop values. And you also didn't add it at the end of this expression.

Reply:
https://github.com/vuetifyjs/vuetify/blob/next/packages/vuetify/src/components/VBanner/VBanner.tsx#L80 was my inspiration.